packages/steps/src/LionSteps.js

```diff
@@ -56,8 +56,13 @@ export class LionSteps extends ObserverMixin(LionLitElement) {
   }
 
   get steps() {
-    let steps = this.children;
-    steps = Array.from(steps).filter(step => step.nodeName === 'LION-STEP');
+    const slots = this.shadowRoot.querySelectorAll('slot');
+    const slot = Array.from(slots).filter(element => element.name === '')[0];
+    if (!slot) {
+      return [];
+    }
+    let steps = slot.assignedNodes();
+    steps = steps.filter(step => step.nodeType !== 3);
```

Comment:
first of all, it is better to use `Node.TEXT_NODE` instead of a magical number

second, the right logic will be `steps.filter(step => step.nodeType === Node.ELEMENT_NODE);`, because you need only element nodes (you don't need comments, right? so just ignoring text nodes is not enough)

third you are filtering nodes, not steps, while trying to find steps, so better to use `node` as an argument name, because it is more precise `nodes.filter(node => node.nodeType === Node.ELEMENT_NODE);`

and in general it is better not to use `let` unless very necessary, and here it can be just a one liner

I would write the entire getter like this:

```js
  get steps() {
    const slot = this.shadowRoot.querySelector('slot[name=""]');
    return slot.assignedNodes().filter(node => node.nodeType === Node.ELEMENT_NODE);
  }
```

Reply:
this is no longer applicable due to the new implementation of the method